db/change_cache.go

```diff
@@ -176,24 +176,20 @@ func (c *changeCache) Init(logCtx context.Context, dbcontext *DatabaseContext, n
 		c.options = DefaultCacheOptions()
 	}
 
-	channelCache, err := NewChannelCacheForContext(c.options.ChannelCacheOptions, c.context)
-	if err != nil {
-		return err
-	}
 	c.channelCache = channelCache
 
-	base.InfofCtx(c.logCtx, base.KeyCache, "Initializing changes cache for database %s with options %+v", base.UD(dbcontext.Name), c.options)
+	base.InfofCtx(c.logCtx, base.KeyCache, "Initializing changes cache for collection %s.%s with options %+v", base.UD(collection.ScopeName()), base.UD(collection.Name()), c.options)
```

Comment:
I think we still want database name in this style of logging (along with scope and collection), as in theory we could have two databases targeting 'bucket1.scope1.collection1' and 'bucket2.scope1.collection1' and we'd want to be able to differentiate the logging.